packages/migrate/index.ts

```diff
@@ -152,19 +167,20 @@ function runMigration(currentConfigPath, outputConfigPath) {
 						console.log(chalk.red("✖ Migration aborted"));
 					}
 				})
-				.then(answer => {
+				.then((answer: object) => {
 					if (!answer) return;
 
-					runPrettier(outputConfigPath, result, err => {
+					runPrettier(outputConfigPath, result, (err: object) => {
 						if (err) {
 							throw err;
 						}
 					});
 
 					if (answer["confirmValidation"]) {
-						const webpackOptionsValidationErrors = validate(
+						const webpackOptionsValidationErrors: any = validate(
```

Comment:
Use of any

Reply:
I think we didn't decide if we can or cannot use `any`. Also, I think we shouldn't enforce types completely. Leaving types at some places implicitly means `any`.